testutil/apparmor.go

```diff
@@ -0,0 +1,78 @@
+package testutil
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+)
+
+func AppArmorParseAndHashHelper(profile string) (string, error) {
+
```

Comment:
Stray newline again?

Reply:
Done in commit 25f75be5889fc9b06f51856734f4fb712dfe42fa